hub/core/meta/encode/creds.py

```diff
@@ -13,9 +13,11 @@ def __init__(self):
         self.is_dirty = False
         super().__init__()
 
-    def register_samples(self, item: Any, num_samples: int):
+    def register_samples(
+        self, item: Any, num_samples: int, end: bool = True, row: Optional[int] = None
+    ):
         self.is_dirty = True
-        return super().register_samples(item, num_samples)
+        return super().register_samples(item, num_samples, end=end, row=row)
```

Comment:
why are we doing this? creds encoder is not on chunk level but rather on tensor level, it doesn't need end or row arguments